cuegui/cuegui/ServiceDialog.py

```diff
@@ -262,14 +259,16 @@ def refresh(self):
                 self.__services = opencue.api.getDefaultServices()
             else:
                 self.__services = self.__show.getServiceOverrides()
-        except Exception:
-            return
+        except Exception as e:
+            raise Exception(e)
 
         for service in self.__services:
-            item = QtWidgets.QListWidgetItem(service.name())
+            item = QtWidgets.QListWidgetItem(service.data.name)
```

Comment:
Any reason for `.data.name` over `.name()`? In general we prefer to use the wrapper methods over accessing `data` directly.